flowkit/flowkit.go

```diff
@@ -307,8 +307,6 @@ func (f *Flowkit) AddContract(
 		return flow.EmptyID, false, err
 	}
 
-	f.logger.StartProgress(fmt.Sprintf("Checking contract '%s' on account '%s'...", name, account.Address))
```

Comment:
I don't like this being removed, it provides valuable progress to the user while the update is happening.

Reply:
What I believe it should be done is for the diff prompt to use the same logger, we should never use fmt.Println to output data, just because of this kind of issues. So make sure to use the logger here https://github.com/onflow/flow-cli/blob/master/internal/util/prompt.go#L257 instead of fmt.Println. That way the logger will hide the progress log.

Reply:
Changed